pom.xml

```diff
@@ -326,7 +326,7 @@
                     <dependency>
                         <groupId>com.google.errorprone</groupId>
                         <artifactId>error_prone_core</artifactId>
-                        <version>2.1.2</version>
+                        <version>2.1.3</version>
```

Comment:
@joschi The Jenkins build fails with the following message:

```
[ERROR] COMPILATION ERROR : 
[INFO] -------------------------------------------------------------
[ERROR] /var/lib/jenkins/workspace/graylog-project-pr-snapshot/.repos/graylog-plugin-cef/src/test/java/org/graylog/plugins/cef/codec/CEFCodecFixturesTest.java:[82,47] [StreamResourceLeak] Streams that encapsulate a closeable resource should be closed using try-with-resources
    (see http://errorprone.info/bugpattern/StreamResourceLeak)
  Did you mean 'final File[] fixtureFiles ;'?
```

So we either have to create a PR for the CEF plugin or disable the rule.

Reply:
https://github.com/Graylog2/graylog-plugin-cef/pull/25